### PR TITLE
Add Task1.py

### PR DESCRIPTION
**Pull request by Drelarusoidd**

### CODE REVIEW

File: Tasks/Cheretun/task1.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cash = 20000
+month = 5 * 12
+percent = 15 / 100
+
+print(round(cash*(1+ percent / 12)**month, 3))
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Cheretun/task1.py:5]: why rounding to the 3rd sign? just why???

File: Tasks/Cheretun/task/cubic equation.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from math import *
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Cheretun/task/cubic equation.py:1]: you're totally gonna overwrite something with such imports)

**Review by MikitaTsiarentsyeu — APPROVED** [review]
The main task is a little bit sloppy, the cubic eq is ok. The score is 9.

File: Tasks/Cheretun/task1.py (hunk 1 above)
Comment (Drelarusoidd) [reply Tasks/Cheretun/task1.py:5]: I don't know , some mistake really.

File: Tasks/Cheretun/task/cubic equation.py (hunk 2 above)
Comment (Drelarusoidd) [reply Tasks/Cheretun/task/cubic equation.py:1]: ok understood

**Pull request merged**